src/modeler/AutoRest.Swagger/Validation/AvoidMsdnReferences.cs

```diff
@@ -11,6 +11,11 @@ namespace AutoRest.Swagger.Validation
 {
     public class AvoidMsdnReferences : TypedRule<string>
```

Comment:
```ValidationChangesImpact``` we can handle in the next rev. But please make sure you make ```ValidationCategory``` and ```Severity``` fields virtual so that every single rule that inherits from ```Rule``` class has this overridden.